content/influxdb/v2.1/reference/syntax/line-protocol.md

```diff
@@ -26,43 +26,59 @@ It is a text-based format that provides the measurement, tag set, field set, and
 - [Comments](#comments)
 - [Naming restrictions](#naming-restrictions)
 - [Duplicate points](#duplicate-points)
+- [Out of range values](#out-of-range-values)
+- [Parse errors](#parse-errors)
 
-```js
-// Syntax
+#### Syntax
+
+```py
+# Line protocol syntax
 <measurement>[,<tag_key>=<tag_value>[,<tag_key>=<tag_value>]] <field_key>=<field_value>[,<field_key>=<field_value>] [<timestamp>]
+```
+#### Example
 
-// Example
+```py
+# Line protocol example
 myMeasurement,tag1=value1,tag2=value2 fieldKey="fieldValue" 1556813561098000000
 ```
 
-Lines separated by the newline character `\n` represent a single point
-in InfluxDB. Line protocol is whitespace sensitive.
+## Lines
 
-{{% note %}}
-Line protocol does not support the newline character `\n` in tag or field values.
-{{% /note %}}
+Line protocol consists of zero or more lines, each terminated by the newline character (`\n`).
+A line may represent one of the following:
+- [data point](#elements-of-line-protocol) that represents a [point](/influxdb/v2.1/reference/key-concepts/data-elements) in InfluxDB
+- blank line that consists entirely of [whitespace](#whitespace)
+- [comment](#comments) that starts with `#`
+
+Line protocol is [whitespace-sensitive](#whitespace).
+Line protocol string elements may support certain [special characters](#special-characters).
 
 ## Elements of line protocol
 
-```
+Lines that represent data points contain a [measurement name](#measurement), optional [tags](#tag-set), at least one [field](#field-set), and an optional [timestamp](#timestamp).
+
+```js
 measurementName,tagKey=tagValue fieldKey="fieldValue" 1465839830100400200
 --------------- --------------- --------------------- -------------------
        |               |                  |                    |
   Measurement       Tag set           Field set            Timestamp
 ```
 
 ### Measurement
+
 ({{< req >}})
 The measurement name.
 InfluxDB accepts one measurement per point.
 _Measurement names are case-sensitive and subject to [naming restrictions](#naming-restrictions)._
 
 _**Data type:** [String](#string)_
 
+To learn more about designing efficient measurements, see [best practices for schema design](/influxdb/v2.1/write-data/best-practices/schema-design/).
 
 ### Tag set
+
 _**Optional**_ –
-All tag key-value pairs for the point.
+All [tag]() key-value pairs for the point.
 Key-value relationships are denoted with the `=` operand.
 Multiple tag key-value pairs are comma-delimited.
 _Tag keys and tag values are case-sensitive.
```

Comment:
The naming restrictions linked below should mention that a tag key or value cannot end with a backslash.